oauth2/oauth2-resource-server/src/test/java/org/springframework/security/oauth2/server/resource/authentication/JwtAuthenticationTokenTests.java

```diff
@@ -98,6 +107,19 @@ public void constructorWhenUsingOnlyJwtThenConstructedCorrectly() {
 		assertThat(token.isAuthenticated()).isFalse();
 	}
 
+	@Test
```

Comment:
Would you please add a test that shows that when using the `(Jwt, Collection)` constructor (this is the one most folks are using in 5.1) continues to work correctly, e.g. `getName()` shows in that case whatever `jwt.getSubject()` returns?